tests/slow/src/test/scala/tests/feature/DocumentSymbolCrossLspSuite.scala

```diff
@@ -0,0 +1,31 @@
+package tests.feature
+
+import tests.BaseLspSuite
+import scala.meta.internal.metals.{BuildInfo => V}
+
+class DocumentSymbolCrossLspSuite extends BaseLspSuite("documentSymbol-cross") {
+
+  test("scala3-syntax-render") {
+    for {
+      _ <- server.initialize(
+        s"""|
+            |/metals.json
+            |{
+            |  "a": { 
+            |    "scalaVersion": "${V.scala3}"
+            |  }
+            |}
+            |/a/src/main/scala/a/Main.scala
+            |type X = String | Int
+            |""".stripMargin
+      )
+      _ <- server.didOpen("a/src/main/scala/a/Main.scala")
+      symbols <- server.documentSymbols("a/src/main/scala/a/Main.scala")
+      _ = assertDiffEqual(
+        symbols.trim,
+        "/*X(TypeParameter):1*/type X = String | Int"
```

Comment:
We could add it to `input3` and just have it tested there. I don't think there is a need for slow test here.